packages/backend/src/routes/CommentRouter.ts

```diff
@@ -246,3 +254,142 @@ async function createCommentSubsidy(req, res) {
         comment,
     })
 }
+
+async function editComment(req, res) {
+    const id = req.params.id
+    const unirepSocialContract = new ethers.Contract(
+        UNIREP_SOCIAL,
+        UNIREP_SOCIAL_ABI,
+        DEFAULT_ETH_PROVIDER
+    )
+
+    // Parse Inputs
+    const { publicSignals, proof, content } = req.body
+    const epkProof = new EpochKeyProof(
+        publicSignals,
+        formatProofForSnarkjsVerification(proof)
+    )
+    const newHashedContent = ethers.utils.keccak256(
+        ethers.utils.toUtf8Bytes(content)
+    )
+
+    const error = await verifyEpochKeyProof(req.db, epkProof)
```

Comment:
We need to check that the `epoch` of the proof is the same epoch that the comment was created. We already store this on the object.